Translate legal (#258)
wyrmworkspublishing/free5e: de-DE/Codex_der_Charaktere/Rechtliches.md

Das BFRD 1.0 ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.

Der Text des Free5e Character's Codex © 2025 von Wyrmworks Publishing, verfügbar unter [https://free5e.com](https://free5e.com), ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International.
Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de).

Du darfst diesen Inhalt in jeder von dieser Lizenz erlaubten Weise verwenden, solange du die folgende Attributionserklärung in dein eigenes Werk aufnimmst:
> Dieses Werk enthält Material, das aus dem **Free5e Character's Codex** adaptiert wurde, © 2025 von Wyrmworks Publishing, verfügbar unter [https://free5e.com](https://free5e.com).
> Der **Free5e Character's Codex** ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International (CC-BY-4.0).

> > Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de).

Author: Agreed. We can pull this discussion out into discord.

> > Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de).

Author: To speed up the process and as we already have an agreement I already applied the changes.

# Rechtliches

> Dieses Werk enthält Material aus dem A5E System Reference Document (A5ESRD) von EN Publishing, verfügbar unter [A5ESRD.com](https://A5ESRD.com), basierend auf Level Up: Advanced 5th Edition, verfügbar unter [www.levelup5e.com](https://www.levelup5e.com).
> Das A5ESRD ist lizenziert unter der Creative Commons Attribution 4.0 International License, verfügbar unter [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).

Reviewer: Um das an die anderen anzupassen:
```suggestion
Dieses Werk enthält Material aus dem A5E System Reference Document ("A5ESRD") von EN Publishing, das unter [A5ESRD.com](https://A5ESRD.com) verfügbar ist.
Das A5ESRD ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.
```

> Dieses Werk enthält Material aus dem System Reference Document 5.1 ("SRD 5.1") von Wizards of the Coast LLC, verfügbar unter [https://dnd.wizards.com/resources/systems-reference-document](https://dnd.wizards.com/resources/systems-reference-document).
> Das SRD 5.1 ist lizenziert unter der Creative Commons Attribution 4.0 International License, verfügbar unter [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).

Reviewer: Wollen wir hier nicht das deutsche Systemreferenzdokument referenzieren?
```suggestion
Dieses Werk enthält Material aus dem Systemreferenzdokument 5.1 ("SRD 5.1") von Wizards of the Coast LLC, das unter [https://dnd.wizards.com/de/resources/systems-reference-document](https://dnd.wizards.com/de/resources/systems-reference-document) verfügbar ist.
Das SRD 5.1 ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.
```
Das ist der Text aus dem deutschen SRD.

> Dieses Werk enthält Material aus dem Lazy GM's Resource Document von Michael E. Shea von [SlyFlourish.com](https://SlyFlourish.com), verfügbar unter der Creative Commons Attribution 4.0 International License.
> <!-- TODO Add a link to the license -->

Reviewer: Wo wir grad dabei sind: Das ist die gleiche Lizenz wie oben auch. Also:
```suggestion
Dieses Werk enthält Material aus dem Lazy GM's Resource Document von Michael E. Shea, das unter [SlyFlourish.com](https://SlyFlourish.com) verfügbar ist.
Das Lazy GM's Resource Document ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.
```

> Dieses Werk enthält Inhalte aus Kibbles' Compendium of Legends and Legacies von KibblesTasty Homebrew LLC, verfügbar unter [https://www.kthomebrew.com/krd](https://www.kthomebrew.com/krd).
> Das Kibbles' Compendium of Legends and Legacies ist lizenziert unter der Creative Commons Attribution 4.0 International License (CC-BY-4.0), verfügbar unter [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).

Reviewer: Um es an die anderen anzupassen:
```suggestion
Dieses Werk enthält Material aus dem Kibbles' Compendium of Legends and Legacies von KibblesTasty Homebrew LLC, das unter [https://www.kthomebrew.com/krd](https://www.kthomebrew.com/krd) verfügbar ist.
Das Kibbles' Compendium of Legends and Legacies ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.
```

> Dieses Werk enthält Material aus dem Black Flag Reference Document 1.0 ("BFRD 1.0") von Kobold Press, verfügbar unter [https://koboldpress.com/Black-Flag-Roleplaying](https://koboldpress.com/Black-Flag-Roleplaying).
> Das BFRD 1.0 ist lizenziert unter der Creative Commons Attribution 4.0 International License, verfügbar unter [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).

Reviewer: Um es an die anderen anzupassen:
```suggestion
Dieses Werk enthält Material aus dem Black Flag Reference Document 1.0 ("BFRD 1.0") von Kobold Press, das unter [https://koboldpress.com/Black-Flag-Roleplaying](https://koboldpress.com/Black-Flag-Roleplaying) verfügbar ist.
Das BFRD 1.0 ist lizenziert gemäß der Lizenz Creative Commons Namensnennung 4.0 International, die unter [https://creativecommons.org/licenses/by/4.0/legalcode.de](https://creativecommons.org/licenses/by/4.0/legalcode.de) verfügbar ist.
```

> Der Text des Free5e Character's Codex © 2025 von Wyrmworks Publishing, verfügbar unter [https://free5e.com](https://free5e.com), ist lizenziert unter Creative Commons Attribution 4.0 International.
> Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).

Reviewer: Hier sollten wir auf die deutsche Version verweisen, also:
```suggestion
Der Text des Free5e Character's Codex © 2025 von Wyrmworks Publishing, verfügbar unter [https://free5e.com](https://free5e.com), ist lizenziert unter Creative Commons Namensnennung 4.0 International.
Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/deed.de](https://creativecommons.org/licenses/by/4.0/deed.de).
```

Du darfst diesen Inhalt in jeder von dieser Lizenz erlaubten Weise verwenden, solange du die folgende Attributionserklärung in dein eigenes Werk aufnimmst:
> Dieses Werk enthält Material, das aus dem **Free5e Character's Codex** adaptiert wurde, © 2025 von Wyrmworks Publishing, verfügbar unter [https://free5e.com](https://free5e.com).

> > Der **Free5e Character's Codex** ist lizenziert unter der Creative Commons Attribution 4.0 International License (CC-BY-4.0).
> > Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).

Reviewer: ```suggestion
> Der **Free5e Character's Codex** ist lizenziert unter der Creative Commons Namensnennung 4.0 International Lizenz (CC-BY-4.0).
> Um eine Kopie dieser Lizenz einzusehen, besuche [https://creativecommons.org/licenses/by/4.0/deed.de](https://creativecommons.org/licenses/by/4.0/deed.de).
```

>
> Diese Adaptierung enthält auch Material, das ursprünglich aus folgenden Quellen stammt:
>
> - Dem **A5E System Reference Document (A5ESRD)** von EN Publishing, verfügbar unter [A5ESRD.com](https://A5ESRD.com) und lizenziert unter CC-BY-4.0.

> > - Dem **System Reference Document 5.1 (SRD 5.1)** von Wizards of the Coast LLC, verfügbar unter [https://dnd.wizards.com/resources/systems-reference-document](https://dnd.wizards.com/resources/systems-reference-document) und lizenziert unter CC-BY-4.0.

Reviewer: ```suggestion
> - Dem ** Systemreferenzdokument 5.1 (SRD 5.1)** von Wizards of the Coast LLC, verfügbar unter [https://dnd.wizards.com/de/resources/systems-reference-document](https://dnd.wizards.com/de/resources/systems-reference-document) und lizenziert unter CC-BY-4.0.
```

> - Dem **Lazy GM's Resource Document** von Michael E. Shea von [SlyFlourish.com](https://SlyFlourish.com), lizenziert unter CC-BY-4.0.
> - Dem **Kibbles' Compendium of Legends and Legacies** von KibblesTasty Homebrew LLC, verfügbar unter [https://www.kthomebrew.com/krd](https://www.kthomebrew.com/krd) und lizenziert unter CC-BY-4.0.
> - Dem **Black Flag Reference Document 1.0 ("BFRD 1.0")** von Kobold Press, verfügbar unter [https://koboldpress.com/wp-content/uploads/2025/07/Black-Flag-Roleplaying-v04_2025_07_01.pdf](https://koboldpress.com/wp-content/uploads/2025/07/Black-Flag-Roleplaying-v04_2025_07_01.pdf) und lizenziert unter CC-BY-4.0.

> Alle in diesem Buch enthaltenen Illustrationen sind lizenziert unter [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) oder [CC-0](https://creativecommons.org/publicdomain/zero/1.0/deed).

Reviewer: ```suggestion
Alle in diesem Buch enthaltenen Illustrationen sind lizenziert unter [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/deed.de) oder [CC-0](https://creativecommons.org/publicdomain/zero/1.0/deed.de).
```

Um diese Illustrationen außerhalb dieses Projekts zu verwenden, siehe das begleitende Dokument mit spezifischen Lizenz- und Quellenangaben.
Falls du diese Kopie ohne jenes Dokument erhalten hast, kannst du eine Kopie unter [https://free5e.com](https://free5e.com) erhalten.

Die [Atkinson Hyperlegible™](https://www.brailleinstitute.org/freefont/) Schriftfamilie, die in einigen Versionen dieses Buches verwendet wird, ist urheberrechtlich geschützt 2020, Braille Institute of America, Inc. ([https://www.brailleinstitute.org/](https://www.brailleinstitute.org/)), mit reservierten Schriftnamen: "ATKINSON" und "HYPERLEGIBLE".
Atkinson Hyperlegible™ ist lizenziert unter der [SIL Open Font License, Version 1.1](https://openfontlicense.org/open-font-license-official-text/).

> Um eine Kopie dieser Lizenz einzusehen, besuche [https://braileinstitute.app.box.com/s/rin3vzegmcy7sil28yfqslz2r5etv5nl](https://braileinstitute.app.box.com/s/rin3vzegmcy7sil28yfqslz2r5etv5nl).

Reviewer: ```suggestion
Um eine Kopie dieser Lizenz einzusehen (nur in Englisch verfügbar), besuche [https://braileinstitute.app.box.com/s/rin3vzegmcy7sil28yfqslz2r5etv5nl](https://braileinstitute.app.box.com/s/rin3vzegmcy7sil28yfqslz2r5etv5nl).
```

Die [Roboto](https://github.com/googlefonts/roboto-3-classic) Schrift, die in einigen Versionen dieses Buches verwendet wird, ist urheberrechtlich geschützt 2011 The Roboto Project Authors ([https://github.com/googlefonts/roboto-classic](https://github.com/googlefonts/roboto-classic)).
Roboto ist lizenziert unter der [SIL Open Font License, Version 1.1](https://openfontlicense.org/open-font-license-official-text/).

> Um eine Kopie dieser Lizenz einzusehen, besuche [https://raw.githubusercontent.com/googlefonts/roboto-3-classic/refs/tags/v3.011/OFL.txt](https://raw.githubusercontent.com/googlefonts/roboto-3-classic/refs/tags/v3.011/OFL.txt).

Reviewer: ```suggestion
Um eine Kopie dieser Lizenz einzusehen (nur in Englisch verfügbar), besuche [https://raw.githubusercontent.com/googlefonts/roboto-3-classic/refs/tags/v3.011/OFL.txt](https://raw.githubusercontent.com/googlefonts/roboto-3-classic/refs/tags/v3.011/OFL.txt).
```

Author: Overlooked it. Thanks for reminding.
